### ISSUE TITLE
Update to WCAG 2.1 Understanding 1.4.1 not published to live version

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#2269]
Just noticed that https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html still doesn't show the additional notes we added ages ago to https://w3c.github.io/wcag/understanding/use-of-color.html (the extra notes about the 3:1 contrast ratio that were bubbled up into the main understanding doc - see https://github.com/w3c/wcag/pull/1500)

This despite the fact that the understanding document itself seems to have been published/republished quite recently (9 March 2022).

/cc @alastc @michael-n-cooper

**Comment by patrickhlauke** [comment]
assume this is due to different branches being used for the old 2.1 and the new 2.2 stuff, and the change only making it into the 2.2 part?

**Comment by alastc** [comment]
@michael-n-cooper Do you remember when the 2.1 branch was separated? #1500 seems like it would be after that?

Could you check the [closed PRs labelled with 2.1](https://github.com/w3c/wcag/pulls?q=is%3Apr+is%3Aclosed+label%3A%22WCAG+2.1%22+)?

**Comment by michael-n-cooper** [comment]
I think #2279 addresses the questions in this thread. If that is incorrect, let me know specifically what I should be looking for.

**Comment by alastc** [comment]
HI @michael-n-cooper, I don't think 2279 addresses this, that was based on the transition period. The changes in this issue are from #1500, from Dec 2020.

I think the common factor is me merging things in without (knowing how to) cherry pick into 2.1 as well. So the bottom of [the list](https://github.com/w3c/wcag/pulls?q=is%3Apr+is%3Aclosed+label%3A%22WCAG+2.1%22+) is #1371, which doesn't appear to be included in https://w3c.github.io/wcag/understanding/contrast-minimum (which still has "criteria" instead of "criterion").

**Comment by patrickhlauke** [comment]
the original problem (discrepancy between https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html and https://w3c.github.io/wcag/understanding/use-of-color.html) is still present

**Comment by michael-n-cooper** [comment]
I found an issue on my system causing the updates to go to the wrong location sometimes. I'll fix that, meanwhile have posted what I think is correct, anyways, the above looks fixed.
